productcomments.php

```diff
@@ -896,6 +896,31 @@ public function hookDisplayFooterProduct($params)
         return $this->renderProductCommentsList($params['product']) . $this->renderProductCommentModal($params['product']);
     }
 
+    /**
+     * Inject data about productcomments in the product object for frontoffice
+     *
+     * @param array $params
+     *
+     * @return void
+     */
+    public function hookFilterProductContent(array $params)
+    {
+        if (empty($params['object']) || empty($params['object']->id)) {
```

Comment:
```suggestion
        if (empty($params['object']->id)) {
```

Should be enough afaik

Reply:
Fixed